### DISCUSSION TITLE
在编辑器点运行如果ts的代码有问题编辑器容易crash并且没有任何提示

### DISCUSSION THREAD

**Discussion by stevenBZ** [Tencent/puerts#1167] [General]
请问有没有办法防止因为ts代码的问题导致编辑器crash，或者crash的时候能不能给提示。

UE编辑器版本 4.27

**Comment by chexiongsheng** [comment]
确认是puerts引起crash就发栈提issue，零容忍。
我没听说这样的反馈，包括内部一些大型项目。按你说的情况的话，我一天到晚被投诉了。
